website/www/site/content/en/documentation/io/built-in/google-bigquery.md

```diff
@@ -323,6 +323,11 @@ in the following example:
 {{< code_sample "sdks/python/apache_beam/examples/snippets/snippets.py" model_bigqueryio_read_query_std_sql >}}
 {{< /highlight >}}
 
+{{< paragraph class="language-py" >}}
```

Comment:
The new render out now looks good. Though, should it be in class="language-java" (or there is a python counterpart) ? And I find switching the language tab this paragraph is always there, not sure why though